test/test-MSONUtility.cc

```diff
@@ -187,6 +187,24 @@ TEST_CASE("Parse required type attribute", "[mson][utility]")
     REQUIRE((typeAttributes & FixedTypeAttribute) == 0);
     REQUIRE((typeAttributes & SampleTypeAttribute) == 0);
     REQUIRE((typeAttributes & DefaultTypeAttribute) == 0);
+    REQUIRE((typeAttributes & NullableTypeAttribute) == 0);
+    REQUIRE(isAttributeParsed);
+}
+
+TEST_CASE("Parse nullable type attribute", "[mson][utility]")
```

Comment:
Looks great. Can you move this test case to be after https://github.com/apiaryio/snowcrash/pull/349/files#diff-541e593511598884d82ffa9b97eecc95R277?